lib/movie.dart

```diff
@@ -0,0 +1,34 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String overview;
+  late DateTime releaseDate;
```

Comment:
if this value its not defined yet, you can assign it as final nullable

Reply:
Done! I made it a final attribute, it was a late attribute because of a previous try of how implement it